add a fido2 guide (#17261)
DataDog/documentation: content/en/synthetics/settings/_index.md

1. In **Choose variable type**, select **MFA Token**.
2. In **Define Variable**, enter a **Variable Name**. Your variable name can only use uppercase letters, numbers, and underscores.
3. Optionally, enter a **Description** and select **Tags** to associate with your variable. 
4. Enter the **Secret Key** to your variable or upload a QR code image.
5. Click **+ Generate** to create an OTP. You can copy the generated OTP with the **Copy** icon.

{{< img src="synthetics/guide/browser-tests-totp/new-variable-totp.png" alt="Create a MFA token" style="width:100%;" >}}

For more information about TOTP-based MFA in a browser test, see [TOTPs For Multi-Factor Authentication (MFA) In Browser Tests][1].

[1]: /synthetics/guide/browser-tests-totp
{{% /tab %}}

> {{% tab "Virtual Authenticator" %}}

Author: @ruizb 
Get you please give a quick review for the section below

Reviewer: Do we want to keep "virtual authenticator" in the docs, despite using "FIDO2" in the UI?

Author: we changed the UI to have Virtual Auth as GV name

> To complete a user journey with a passkey in your Synthetics tests, create a Virtual Authenticator global variable. This global variable will generate and store passkeys for all your Synthetics browser tests. For more information, see [Using passkeys In Browser Tests][1].

Reviewer: ```suggestion
To complete a user journey with a passkey in your Synthetics tests, create a Virtual Authenticator global variable. This global variable is used to generate and store passkeys for all your Synthetics browser tests. For more information, see [Using passkeys In Browser Tests][1].
```

1. Navigate to the **Global Variables** tab in [**Synthetic Monitoring & Continuous Testing** > **Settings**][1] and click **+ New Global Variable**.

1. In the **Choose variable type** section, select **Virtual Authenticator**.
2. In the **Specify variable details** section, enter a **Variable Name**. Your variable name can only use uppercase letters, numbers, and underscores.

> 3. Optionally, enter a **Description** and select **Tags** to associate with your variable. Datadog then creates a virtual authenticator that will generate and store your passkeys. 

Reviewer: ```suggestion
3. Optionally, enter a **Description**, and select **Tags** to associate with your variable. Datadog then creates a virtual authenticator used to generate and store your passkeys.
```

4. In the **Permissions settings** section, restrict access to your variable based on roles in your organization. For more information about roles, see the [RBAC documentation][2].

> {{< img src="synthetics/guide/browser-tests-passkeys/new-variable-fido2.png" alt="Create a FIDO2 key" style="width:80%;" >}}

Author: screenshot will be updated once app work is done
